github/ci/prow-deploy/files/jobs/RHsyseng/rhcos-slb/rhcos-slb-presubmits.yaml

```diff
@@ -0,0 +1,35 @@
+---
+presubmits:
+  RHsyseng/rhcos-slb:
+    - name: pull-rhcos-slb-ci-network-tests-main
+      branches:
+        - main
+      annotations:
+        fork-per-release: "true"
+      always_run: false
+      optional: true
+      decorate: true
+      cluster: prow-workloads
+      decoration_config:
+        timeout: 30m
+        grace_period: 5m
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20210706-a5d2c6b
+            securityContext:
+              privileged: true
```

Comment:
Does the job need to run in privileged mode?

Reply:
only for the pre-requisites installation part

Reply:
I think privileged in this context is not related to executing as root, but instead to have access to the host's devices, which for this lane is not needed right?

In any case you will execute as root in a container built from `quay.io/kubevirtci/bootstrap:v20210706-a5d2c6b`, no matter if it is privileged or not.

Reply:
commented on this [here](https://github.com/kubevirt/project-infra/pull/1423#issuecomment-879694635)